scripts/generate

```diff
@@ -209,7 +277,12 @@ const generate =
 def('generate',
     {},
     [$.String, $.String],
-    pipe([replace(/<h4 name=/g, '<h4 id='),
+    pipe([replace_(regex('g', '<h4 name="(.*)"><code><a href="(.*)">([^ ]*)(.*)</a></code></h4>'),
+                   ([id, href, name, _innerHtml]) =>
+                     el('h4', {id},
+                        el('code', {},
+                           el('a', {href}, name)
+                           + j(map(linkTokens, split(/(\w+)/, _innerHtml)))))),
```

Comment:
This is reminiscent of [Hiccup][1]. :)


[1]: https://github.com/weavejester/hiccup